be/src/exec/vectorized/schema_scan_node.cpp

```diff
@@ -174,87 +178,77 @@ Status SchemaScanNode::get_next(RuntimeState* state, ChunkPtr* chunk, bool* eos)
     const std::vector<SlotDescriptor*>& src_slot_descs = _schema_scanner->get_slot_descs();
     const std::vector<SlotDescriptor*>& dest_slot_descs = _dest_tuple_desc->slots();
 
-    *chunk = std::make_shared<vectorized::Chunk>();
-    if (nullptr == chunk->get()) {
+    bool scanner_eos = false;
+    int row_num = 0;
+
+    ChunkPtr chunk_src = std::make_shared<vectorized::Chunk>();
+    if (nullptr == chunk_src.get()) {
```

Comment:
suggest to invoke chunk_src's reserve method to make room for it.

Reply:
done